src/sidebar/components/test/annotation-body-test.js

```diff
@@ -65,7 +65,10 @@ describe('AnnotationBody', () => {
     const button = wrapper.find('Button');
     assert.isOk(button.exists());
     assert.equal(button.props().buttonText, 'More');
-    assert.equal(button.props().title, 'Toggle to show full annotation text');
+    assert.equal(
```

Comment:
I see that `aria-label` is not tested here, but is in excerpt.js. I think this is okay as long as we have axe coverage.